Preferences Page

Preferences page suffers a provider bug in need of a check: 
`bundle.js:201129 Error: [$injector:unpr] Unknown provider: APP_PREF_SCHEMAProvider <- APP_PREF_SCHEMA <- UserService`

--- a/public/app/services/user-service/index.js
+++ b/public/app/services/user-service/index.js
@@ -2,4 +2,4 @@ import angular from 'angular';
 import UserService from './user.service';
 
 export default angular.module('hepicApp.services.user', [])
-  .factory('UserService', /* @ngInject */ ($http, API, APP_PREF_SCHEMA) => new UserService($http, API, APP_PREF_SCHEMA));
+  .factory('UserService', /* @ngInject */ ($http, API) => new UserService($http, API));

--- a/public/app/services/user-settings-service/index.js
+++ b/public/app/services/user-settings-service/index.js
@@ -2,4 +2,4 @@ import angular from 'angular';
 import UserSettingsService from './user-settings.service';
 
 export default angular.module('hepicApp.services.user-settings', [])
-  .factory('UserSettingsService', /* @ngInject */ ($http, API, APP_PREF_SCHEMA) => new UserSettingsService($http, API, APP_PREF_SCHEMA));
+  .factory('UserSettingsService', /* @ngInject */ ($http, API) => new UserSettingsService($http, API));

